doc/howto/instances_create.md

```diff
@@ -50,16 +50,20 @@ The following examples use `lxc launch`, but you can use `lxc init` in the same
 
 ### Launch a container
 
-To launch a container with a Ubuntu 22.04 image from the `images` server using the instance name `ubuntu-container`, enter the following command:
+To launch a container with an Ubuntu 22.04 image from the `images` server using the instance name `ubuntu-container`, enter the following command:
 
     lxc launch images:ubuntu/22.04 ubuntu-container
```

Comment:
Please could you update the examples to use `ubuntu:22.04`?

Reply:
Sure but if the reason was due to cloud-init resizing the rootfs automatically, that's no longer needed as even regular (!= cloud) ones from `images:` have auto partition/FS resize now.

Reply:
No, my understanding was that the docs should be using the official ubuntu images.